tests/pf_gp_exceptions/enc/enc.c

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/tests.h>
+#include "pf_gp_exceptions_t.h"
+
+uint64_t faulting_address;
+uint32_t error_code;
+
+uint64_t test_pfgp_handler(oe_exception_record_t* exception_record)
+{
+    if (exception_record->code == OE_EXCEPTION_PAGE_FAULT)
+    {
+        faulting_address = exception_record->faulting_address;
+        error_code = exception_record->error_code;
+        exception_record->context->rip += 3;
```

Comment:
3 looks like a magic number here.  Either add a #define for it, or add a comment explaining where the number comes from.